changelogs/current.yaml

```diff
@@ -14,6 +14,9 @@ behavior_changes:
     :ref:`initial_fetch_timeout <envoy_v3_api_field_config.core.v3.ConfigSource.initial_fetch_timeout>` times out, and will then apply
     the cached assignment and finish updating the warmed cluster. This change is disabled by default, and can be enabled by setting
     the runtime flag ``envoy.restart_features.use_eds_cache_for_ads`` to true.
+- area: http
+  change: |
+    Flip default value of ``envoy.reloadable_features.http1_use_balsa_parser`` runtime flag from false to true.
```

Comment:
I think we want this more informative.
"Switching from http_parser to balsa_parser for handling HTTP/1.1 traffic.  See [issue url] for details.  this behavioral change can be reverted by setting  ``envoy.reloadable_features.http1_use_balsa_parser``  to false.